CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## v1.0.3
+
+*Jan 20, 2022*
+* Improve p2p/peer reactor
+
+### BREAKING CHANGE
+- Nothing
+
+### FEATURES
+- Nothing
+
+### IMPROVEMENTS
+- [p2p/peer] [\#341](https://github.com/line/ostracon/pull/341) Remove default case
+- [review] [\#346](https://github.com/line/ostracon/pull/346) Add CODEOWNERS
+- [upgrade] [\#349](https://github.com/line/ostracon/pull/349) Main patch from tm-v0.34.9
```

Comment:
I prefer to use `backport/tendermint` or `bp/tendermint`. How about other CODEOWNERS?

Reply:
I think `backport/tendermint` is better, too.